### PR TITLE
Vesuvio - VesuvioCorrections test to system test from unit test

### LINKED ISSUE #15786: Vesuvio - Convert VesuvioCorrectionsTest to system test
VesuvioCorrectionsTest currently takes far too long to complete due to it's complexity. In order to speed up unit tests this would be better moved to a system test.

### PR DESCRIPTION
**Pull request by ElliotAOram**
The VesuvioCorrections unit test has been changed to be a unit test due to it's long execution time

**To test:**
- Ensure that the VesuvioCorrectionsTest is a system test and passes.
- Ensure that the VesuvioCorrectionsTest (that is a unit test) no longer exists.

Fixes #15786.

_Internal system test change: Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
- [x] Is the code of an acceptable quality?
- [x] Does the code conform to the coding standards? Is it well structured with small focussed classes/methods/functions?
- [x] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?
##### Functional Tests
- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: Testing/SystemTests/tests/analysis/VesuvioTesting.py [hunk 1]
```diff
@@ -0,0 +1,514 @@
+from mantid.simpleapi import *
```
Comment (FedeMPouzols) [inline Testing/SystemTests/tests/analysis/VesuvioTesting.py]: Did you forget to (re-)move VesuvioTesting.py from Framework/PythonInterface...? Or maybe you need it for some other vesuvio algorithms? It wouldn't be nice to have it duplicated.

**Comment by FedeMPouzols** [comment]
This is working well. In the CI builds outputs you can see that `VesuvioCorrections` is no longer in the list of unit tests but it is run as a system test.

File: Testing/SystemTests/tests/analysis/VesuvioTesting.py (hunk 1 above)
Comment (ElliotAOram) [inline Testing/SystemTests/tests/analysis/VesuvioTesting.py]: I did just forget to remove this from `Framework/PythonInterface` but actually on closer inspection it is  required by both. I agree we don't want a duplicate. Do you know what the best practice would be in terms of referencing the VesuvioTesting.py from both system tests and unit tests? Also where it should go if both are using it?

Comment (FedeMPouzols) [inline Testing/SystemTests/tests/analysis/VesuvioTesting.py]: Maybe these could go in scripts/Inelastic/vesuvio together with other common useful functions? Probably @martyngigg has a better idea what's the best place for this.

Comment (martyngigg) [inline Testing/SystemTests/tests/analysis/VesuvioTesting.py]: Yes, I think we could add this to a `testing.py` module of the `vesuvio` package. It shouldn't do any harm.

**Comment by FedeMPouzols** [comment]
The old files are now removed and the common testing functions are now in`scripts/Inelastic/vesuvio/testing.py`. All looks good: :shipit:

**Pull request merged**